src/ios.js

```diff
@@ -0,0 +1,55 @@
+export const iOSversion = () => {
```

Comment:
it seems we don't need the version ? What do you think about something like that ?
```
isIos = () => /(iPhone|iPad|iPod)/.test(navigator.platform)
```

Reply:
Yeah, works for me.  Hopfully Apple will fix it in iOS 14 and we'll have to change it back, but I think that's optimistic.  This has been in there a while.